deployment/environment/memory/chain.go

```diff
@@ -98,35 +96,19 @@ func evmChain(t *testing.T, numUsers int) EVMChain {
 // TODO: add dynamic users and admin like done in evmChain
 func solChain(t *testing.T) (string, string) {
 	t.Helper()
-	port := "8899"
-	portInt, _ := strconv.Atoi(port)
 
-	faucetPort := "8877"
-	url := "http://127.0.0.1:" + port
-	wsURL := "ws://127.0.0.1:" + strconv.Itoa(portInt+1)
-
-	args := []string{
-		"--reset",
-		"--rpc-port", port,
-		"--faucet-port", faucetPort,
-		"--ledger", t.TempDir(),
+	bcInput := &blockchain.Input{
+		Type:      "solana",
+		Image:     "f4hrenh9it/solana",
```

Comment:
We should use the official image: https://hub.docker.com/layers/solanalabs/solana/v1.18.26/images/sha256-098806e64d44bccdbedbf07c2edabd1c850b92c8a4a0f81eba9c789034813db6

It contains `solana-test-validator`

Reply:
Ah, looks like we've used this more recently: https://github.com/smartcontractkit/chainlink-solana/blob/61c33fc0ce3034c08c99d438833228052ed83044/scripts/setup-localnet/localnet.sh#L18-L26